### ISSUE TITLE
Files not be found

### ISSUE THREAD

**Issue by samdhu1116** [rstudio/shiny-examples#59]
I am unable to run the shiny app using the code as i am unable access the web links which are in the code for transit. Can you please help me on this? I have project for i will need this as reference.

This is just amazing to use for my project.

Thanks

**Comment by jcheng5** [comment]
Can you be more specific as to what you're trying to do, what steps you tried, and what went wrong?

**Comment by samdhu1116** [comment]
Hi Joe,

I downloaded the codes from this link*
https://github.com/rstudio/shiny-examples/tree/master/086-bus-dashboard
<https://github.com/rstudio/shiny-examples/tree/master/086-bus-dashboard>.*


[image: Inline image 1]



In the above link there is a folder by name metrotransit-data
<https://github.com/rstudio/shiny-examples/tree/master/086-bus-dashboard/metrotransit-data>
which
contains a file by name 00-fetch-data.R
<https://github.com/rstudio/shiny-examples/blob/master/086-bus-dashboard/metrotransit-data/00-fetch-data.R>
when
i try to run this code i get an error as its unable to download the zipped
file that is mentioned in the code. - I will need your help on this so that
i can use your creation.

Again, when i use the R Shiny code. The below code where we connect to the
svc* metrotransit link*
getMetroData <- function(path) {
* url <- paste0("http://svc.metrotransit.org/NexTrip/
<http://svc.metrotransit.org/NexTrip/>", path, "?format=json")*
* jsonlite::fromJSON(url)*
*} i get the below error [image: Inline image 2] *
I request you to help me resolve this so that i can use the shiny.

Thanks,
Madhuri

**Comment by jcheng5** [comment]
@wch looks like the transit data has moved.

> The google_transit.zip file is no longer being maintained at this site. In the future, please download the Metro Transit Weekly Schedule update in the GTFS format from here:
> 
> https://gisdata.mn.gov/dataset/us-mn-state-metc-trans-transit-schedule-google-fd

**Comment by samdhu1116** [comment]
Thanks for your response.

But its still not working.

**Comment by jcheng5** [comment]
@samdhu1116 Yeah, my message was directed not at you, but at @wch, the author of the app.
